include/envoy/ssl/context_manager.h

```diff
@@ -24,19 +24,30 @@ class ContextManager {
 
   /**
    * Builds a ServerContext from a ServerContextConfig.
+   * The skip_context_update parameter is used for fast-path (avoiding lock & context looup)x
```

Comment:
errant "x"

Reply:
@PiotrSikora You wouldn't happen to use Emacs would you? 😜

Reply:
Done (and no, I don't use Emacs).